controllers/help-requests/help_requests.controller.js

```diff
@@ -202,12 +210,11 @@ module.exports = {
 
         if (!errors.isEmpty()) {
             var extractedErrors = mapFieldErrors(errors);
-
+            let errorMessage = "Please complete all fields marked in red before updating.";
```

Comment:
nit: I'd use language as suggested in [the GOV.UK guidance](https://design-system.service.gov.uk/components/error-message/). 

What we have here is closer to the [error summary component](https://design-system.service.gov.uk/components/error-summary/), as we don't have the tech for individual error messages yet.
```suggestion
            let errorMessage = "Fill in the fields marked in red.";
```

Reply:
Ah, I see this was already the error message before these changes.

I think it's still worth the effort to change whilst we're here, but up to you.

Reply:
no problem, I'll have a go now